### ISSUE TITLE
Installation on Raspberry Pi

### ISSUE THREAD

**Issue by scargill** [monteslu/node-red-contrib-gpio#7]
I followed the instructions to the letter - and installed this on a Raspberry Pi 2.

If I pull in an output node - attach to pin "13" with an alternating test incoming signal - I get flashing on connector pin 21, GPIo9 on the Pi2... not sure what that has to do with 13 ??

However, if I try to add another pin such as 12, or even rename the first one - I get "connecting~" - and it just won't do anything.

Am I missing something??

Pete.

**Comment by monteslu** [comment]
Pin mappings on raspberry pi are a little weird.
Did you take a look at this list:  https://github.com/nebrius/raspi-io/wiki/Pin-Information#model-abraspberry-pi-2

**Comment by scargill** [comment]
Yes that is INDEED helpful - I picked 17 didn't I  - which, strangely does not appear in the diagram you pointed me to!!! So I tried 22 and that worked. 

BUT in order to get more than one pin working or even to amend an existing one - I had to create a NEW configuration for each one - in the case of amending a connection, deleting the node and configuration and recreating it.  Surely that can't be deliberate?

**Comment by scargill** [comment]
Oh dear, and Servo doesn't do anything on a pin (22) that works for digital IO... Fired 30 and 120 - nothing out on the LED.....

**Comment by monteslu** [comment]
You should absolutely not have to create a new board instance for each pin.  Not sure what's going on there. A little tied up, but will see if I can reproduce soon.

For servos and other PWMs you can only use physical pins 35 and 32.
